### DISCUSSION TITLE
How to get a list of Resolvers specified in the .parcelrc

### DISCUSSION THREAD

**Discussion by grahamcracker1234** [parcel-bundler/parcel#7785] [Q&A] [answered]
I would like to make a custom parcel transformer that needs to resolve the file path of certain files. These files are not dependencies. I would like to somehow resolve the file path using the supplied Resolvers specified in the .parcelrc file. 

The reason for this is because I wish the read the text of the resolved file and insert that text directly into the main file as the transform, but I cannot figure out how to have the user specify which resolver they would like to use, rather than hardcoding a specific resolver directly in

**Comment by mischnic** [comment, marked as answer]
The `resolve` parameter available in the `transform` is designed for exactly this situation: https://github.com/parcel-bundler/parcel/blob/c9264c4c5429b03d48b2f7699f511266db324c54/packages/core/types/index.js#L1040
